packages/store/src/store.ts

```diff
@@ -126,7 +127,14 @@ export class Store {
             this._actions,
             action
           )
-          .pipe(map(() => this._stateStream.getValue()));
+          .pipe(
+            tap(() => {
+              this._actions.next({ action, status: ActionStatus.Completed });
```

Comment:
What do you think about making these class and just say: `new CompletedAction(action)`?

Reply:
we could do that